chpl/chpl-service/src/main/java/gov/healthit/chpl/entity/TestFunctionalityCriteriaMapEntity.java

```diff
@@ -0,0 +1,110 @@
+package gov.healthit.chpl.entity;
+
+import java.io.Serializable;
+import java.util.Date;
+
+import javax.persistence.Basic;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "test_functionality_criteria_map")
+public class TestFunctionalityCriteriaMapEntity implements Serializable{
+    private static final long serialVersionUID = 6446486138564063907L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Basic(optional = false)
+    @Column(name = "id", nullable = false)
+    private Long id;
+
+    @Basic(optional = false)
+    @OneToOne(optional = false, fetch = FetchType.LAZY)
+    @JoinColumn(name = "criteria_id", insertable = false, updatable = false)
+    private CertificationCriterionEntity criteria;
+    
+    @Basic(optional = false)
+    @OneToOne(optional = false, fetch = FetchType.LAZY)
+    @JoinColumn(name = "test_functionality_id", insertable = false, updatable = false)
+    private TestFunctionalityEntity testFunctionality;
+    
+    @Basic(optional = false)
+    @Column(name = "creation_date", nullable = false)
```

Comment:
Can you make creation date and last modified date also insertable = false and updatable = false? The older entities aren't written that way but most(?) newer ones are since these values are set by the db and triggers rather than through the API.

Reply:
Done